lib/device/devicebuilder.js

```diff
@@ -716,7 +716,17 @@ const deviceBuilder = class DeviceBuilder {
    * @param {String} configuration.uri HTTP URI pointing to an image resource. JPG and PNG images are supported.
    * @param {String} configuration.size image size in the ui, either 'small' or 'large'. The small image has the size of a button while the large image is a square image using full width of the client.
    * @param {Function} controller A function which returns the address (URL) to the current image.
-   * @description Add an image to your custom element (for example to display the album cover of the current track)
+   * @description Add an image to your custom element (for example to display the album cover of the current track).
+   * To avoid downloading and resizing large images on the Brain use these general guidelines:
+   * - Large images: 480 x 480
+   * - Small: 100 x 100
+   * If you want to optimize for a specific target:
```

Comment:
minor: maybe it's how the doc parser works but looking at the html above it seems like this line (and the `**note**` below) get bundled in to the previous item instead of stopping the list. Maybe an additional empty line is needed before